README.md

```diff
@@ -10,35 +10,59 @@
     </a>
 </p>
 
-
 <p align="center">
 <img src="assets/logo.png" width="350" /></br>
 <h2>Generate distributed traces for any application in k8s without code changes.</h2>
 </p>
 
 <h2 align="center">
-    <a href="https://www.youtube.com/watch?v=nynyV7FC4VI">Demo Video</a> • <a href="https://docs.odigos.io">Documentation</a> • <a href="https://join.slack.com/t/odigos/shared_invite/zt-1d7egaz29-Rwv2T8kyzc3mWP8qKobz~A">Join Slack Community</a>
+    <a href="https://youtu.be/bqTrGi3skhk?si=CLAolYpdT7rQb7PH">Demo Videos</a> • <a href="https://docs.odigos.io">Documentation</a> • <a href="https://join.slack.com/t/odigos/shared_invite/zt-1d7egaz29-Rwv2T8kyzc3mWP8qKobz~A">Join Slack Community</a>
 </h2>
 
+## What is Odigos?
+
+Odigos is an open-source distributed tracing solution that revolutionizes observability for Kubernetes environments. It provides instant tracing capabilities without requiring any code changes to your applications.
+
+## Key Features
+
+* **Code-Free Instrumentation**: Set up distributed tracing in minutes, eliminating manual code modifications.
+* **Multi-Language Support**: Works with Java, Python, .NET, Node.js, and Go applications.
+* **eBPF-Powered**: Utilizes eBPF technology for high-performance instrumentation, especially for Go applications.
+* **OpenTelemetry Compatible**: Generates traces in OpenTelemetry format for broad tool compatibility.
+* **Vendor Agnostic**: Integrates with various monitoring solutions, avoiding vendor lock-in.
+* **Automatic Scaling**: Manages and scales OpenTelemetry collectors based on data volume.
+* **User-Friendly Management**: Offers a web UI for easy configuration and management.
```

Comment:
We can mention that odigos is opinionated, supply common defaults and best practices out-of-the-box without requiring deep knowledge with OpenTelemetry